packages/Confirmation/src/Confirmation.js

```diff
@@ -0,0 +1,147 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Button from "@paprika/button";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import Popover from "@paprika/popover";
+import Trigger from "./components/Trigger";
+import contentStyles from "./ContentStyles.styles";
+
+import {
+  confirmStyles,
+  confirmHeaderStyles,
+  confirmDescriptionStyles,
+  confirmFooterStyles,
+} from "./Confirmation.styles";
+
+const propTypes = {
+  buttonSize: PropTypes.oneOf(ShirtSizes.DEFAULT),
+  confirmButtonType: PropTypes.oneOf(["primary", "destructive"]),
+  confirmLabel: PropTypes.node.isRequired,
+  description: PropTypes.node,
+  heading: PropTypes.node.isRequired,
```

Comment:
Still think this should be a `string` and not `isRequired. Then wrapping it in a `<Heading>` can be taken care of by this component.

Reply:
Ok i'm just rendering it as a string, not as html though. IS that what you mean?